docs/development/pigs/pig-008.rst

```diff
@@ -0,0 +1,594 @@
+evaluation, but split out the background evaluation. Update test and tutorials
+(`#2026 <https://github.com/gammapy/gammapy/issues/2026>`_)
+
+3. Implement the `Datasets` class in `gammapy.utils.fitting.datasets`, add tests
+using multiple `MapDataset`. Change the `Fit` interface to take a `Datasets`
+object or list of `MapDataset` on input (`#2030 <https://github.com/gammapy/gammapy/issues/2030>`_).
+
+4. Enable joint-likelihood analyses by filtering the `Datasets.parameters` for
+unique parameters only. Add a tutorial for joint-likelihood fitting of multiple
+observations (`#2045 <https://github.com/gammapy/gammapy/issues/2045>`_).
+
+5. Implement `MapDataset.setup()` method, using a list of `MapEvaluator`
+objects. Add an `.evaluation_radius()` attribute to all the spatial models.
+Only support a fixed PSF and edisp per model component (`#2071 <https://github.com/gammapy/gammapy/issues/2071>`_).
+
+6. Add support for psf maps to `MapDataset.setup()`. Extend the `.setup()` method
+to look up the correct PSF for the given model component. A `PSFMap` has to be
+passed on `MapDataset.__init__()` (**v0.14**).
+
+7. Add support for energy dispersion maps to `MapDataset`. Extend the `.setup()`
+method to look up the correct EDisp for a given model component. An `EdispMap`
+has to be passed on `MapDataset.__init__()` (**v0.14**).
+
+8. Add tutorial for joint-likelihood fitting of IACT and Fermi-LAT data, based
+on the joint-crab dataset (**v0.14**).
+
+9. Add a `name` attribute to datasets and a `Datasets.__getitem__` method (**v0.14**).
+
+10. Implement dataset serialization to yaml (**v0.14**).
+
+11. Implement datasets serialization to a single fits file (`#2264 <https://github.com/gammapy/gammapy/issues/2264>`_).
+
+12. Add the direct likelihood evaluation to the `FluxPointsDataset`, by
+interpolating flux point likelihood profiles (**v0.14**).
+
+13. Implement `MapDataset.fake()`, `SpectrumDataset.fake()` and `SoectrumDatasetOnOff.fake()` methods (**v0.14**).
+
+14. Implement convenience helper methods `.residuals()`, `.peek()` and `.__str__` for
+all datasets (**v0.14**).
+
+
+Outlook
+=======
+
+Parallel evaluation of datasets
+-------------------------------
+
+For efficient joint-likelihood fits of multiple observations, parallel processing
+should be used. The obvious entry point is to evaluate one dataset per process and
+join the likelihoods at the end. The current handling of model references
+(the same model instance is passed to different datasets to achieve a generic
+parameter linking by updating the model parameters in place), sets a limitation
+on the parallel evaluation, because Python objects can't easily be shared between
+multiple processes. We are aware of this issue, but propose to solve it later,
+because it does not affect the proposed API for datasets.
+
+
+Lazy loading of Datasets
+------------------------
+For the use-case of inspecting or stacking individual datasets (e.g. MapDataset per observation)
```

Comment:
Probably reading of multiple datasets should be moved from the outlook to the main section.
We already had this for spectra - it was in the SpectrumObservationList.
This functionality should be brought back and since it's not obvious how to do it, the proposed plan to re-add it should IMO be mentioned in the PIG.

Personally I'm +1 to introduce a YAML-based format that points to FITS files, and to make this work for multiple spectra somehow (don't care if generated filenames or multiple folders for now) - i.e. propose something that allows us to store between spectrum data reduction and fitting, and then we can improve / refactor serialisation formats in the coming months. But the need for this should be mentioned in the PIG IMO.